### ISSUE TITLE
Order of Assignments on Course Page

### ISSUE THREAD

**Issue by ChasonMindImp** [maltyeva/mindimp-pg#101]
I am adding assignments to the course page for public speaking, and it adds the most recent assignment to the top. My initial reaction/preference would be for new ones to be added below, so it reads from the beginning of the course to the end, like a syllabus would. However, I wanted to turn it over to everyone and see what people think.

**Comment by jzthompson** [comment]
I've been adding assignments to courses right after they are assigned to students. (For example I just assigned pre-class homework for the second orientation class after I taught the first class on Monday.) For me, I actually like having the most recent (current) assignment appear on top. Would there be a way to toggle how the assignments are ordered so that instructors can choose?

**Comment by ChasonMindImp** [comment]
Or maybe even the students? Perhaps a default sort for progression
throughout the class, and then a sort for by due date or something that
puts the most recent ones first?

On Mon, Apr 27, 2015 at 9:07 PM, jzthompson <notifications@github.com>
wrote:

> I've been adding assignments to courses right after they are assigned to
> students. (For example I just assigned pre-class homework for the second
> orientation class after I taught the first class on Monday.) For me, I
> actually like having the most recent (current) assignment appear on top.
> Would there be a way to toggle how the assignments are ordered so that
> instructors can choose?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maltyeva/mindimp-pg/issues/101#issuecomment-96643512>.
>



-- 
Chason Dailey | 戴奇森
Program Manager and Advisor* | **思脉 **MindImp*
https://www.mindimp.com/

**Comment by maltyeva** [comment]
yeah, that's definitely doable. Would you want the default sort to be by the date that it is due or the date that it is created?

**Comment by ChasonMindImp** [comment]
Per your other email, if the plan is to have as much of the info as we can
loaded so that, for future courses and future sessions, we do not have to
reinvent the wheel, then (perhaps self-servingly) methinks sorting it like
a syllabus will be a natural starting place.

On Tue, Apr 28, 2015 at 10:43 AM, maltyeva <notifications@github.com> wrote:

> yeah, that's definitely doable. Would you want the default sort to be by
> the date that it is due or the date that it is created?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maltyeva/mindimp-pg/issues/101#issuecomment-96889971>.
>



-- 
Chason Dailey | 戴奇森
Program Manager and Advisor* | **思脉 **MindImp*
https://www.mindimp.com/

**Comment by maltyeva** [comment]
ok, sorted by due dates! Would you want another sorting option or is this okay?

**Comment by ChasonMindImp** [comment]
Umm... I don't think so. Being able to sort Class 1, Class 2... Class 10
fits what I am looking for and Class 10, Class 9... Class 1 fits Jeff and
what he's doing.

On Tue, Apr 28, 2015 at 11:30 AM, maltyeva <notifications@github.com> wrote:

> ok, sorted by due dates! Would you want another sorting option or is this
> okay?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/maltyeva/mindimp-pg/issues/101#issuecomment-96896043>.
>



-- 
Chason Dailey | 戴奇森
Program Manager and Advisor* | **思脉 **MindImp*
https://www.mindimp.com/

**Comment by maltyeva** [comment]
Actually, let me push the change now so you guys can take a look and see if you want the other option to sort as well. i'm currently working on a big thing -- adding advisors to students -- so i dont know when that code will be pushed and I don't want to mix the two

**Comment by maltyeva** [comment]
@jzthompson now that the assignments are live, could you check out the order and see if you want me to add an alternative as well?

**Comment by jzthompson** [comment]
Sorting by due date is fine with me, but is there a way that we could toggle whether the assignments are organized with the most recent at the top or the most recent at the bottom? This isn't tremendously important to me, but if it isn't too difficult the option would be nice to have.
